src/rgb/contract/amount.rs

```diff
@@ -445,7 +445,7 @@ mod serde_impl {
 mod test {
     use super::super::test_helpers::*;
     use super::*;
-    use crate::strict_encoding::StrictDecode;
+    use crate::strict_encoding::{StrictDecode, StrictEncode};
 
     static AMOUNT: [u8; 43] = [
```

Comment:
Can you pls add comments to which values these amounts are corresponding to?

Reply:
Actually I also forgot what was the value for this amount. I will regenerate the data in the next push.